util/unifyfs/src/unifyfs.c

```diff
@@ -99,8 +99,8 @@ static char* usage_str =
     "  -s, --script=<path>       [OPTIONAL] <path> to custom launch script\n"
     "  -S, --share-dir=<path>    [REQUIRED] shared file system <path> for use by servers\n"
     "  -c, --cleanup             [OPTIONAL] clean up the UnifyFS storage upon server exit\n"
-    "  -i, --stage-in=<path>     [OPTIONAL, NOT YET SUPPORTED] stage in file(s) at <path>\n"
-    "  -o, --stage-out=<path>    [OPTIONAL, NOT YET SUPPORTED] stage out file(s) to <path> on termination\n"
+    "  -i, --stage-in=<path>     [OPTIONAL] stage in manifest file(s) at <path>\n"
+    "  -o, --stage-out=<path>    [OPTIONAL] stage out manifest file(s) at <path>\n"
```

Comment:
I think the --stage-out option has to go to options for the terminate command below.

Reply:
Oh right.  Good point.